src/ghci/mod.rs

```diff
@@ -120,6 +128,8 @@ impl GhciOpts {
             hooks: opts.hooks.clone(),
             restart_globs: opts.watch.restart_globs()?,
             reload_globs: opts.watch.reload_globs()?,
+            stdout_writer: tokio::io::stdout().into_ghci_write(),
+            stderr_writer: tokio::io::stderr().into_ghci_write(),
```

Comment:
So your thinking is here we could have some `--tui` flag or similar and use it to set some alternate writers here?